schema.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE users (
+  name VARCHAR(50) NOT NULL,
+  photo VARCHAR(255),
+  bio VARCHAR(250) NOT NULL,
```

Comment:
- [ ] As required for the project, kindly ensure that your project has `PostgreSql` as setting up database and not `sqlite3`.

- [ ] As required for the project, kindly ensure that there are migration files for creating `users`, `posts`, `comments`, and `likes` tables with correct data such as `column names`, `data type`, `Foreign keys`,... 
To help you I recommend you watch **[this video as an example](https://www.youtube.com/watch?v=wbZ6yrVxScM&t=1959s)** before starting or you can use this **[article as reference](https://guides.rubyonrails.org/getting_started.html#creating-the-blog-application)**. All the things you need are the above links I left you.

- [ **OPTIONAL** ] **README.md** I think it would be better if you also update the `Getting Started` section of your section with the correct and appropriate information needed by users in order to use your project locally such as using the correct project name, how to install bundles, create the database, run the server,... To help you, just refer to this **[gist_file as an example](https://gist.github.com/tresorsawasawa/4cde69cff1b40161b9de380f9891f415)**